tests/phpunit/tests/canonical.php

```diff
@@ -270,10 +270,11 @@ public function test_strict_redirect_guess_404_permalink() {
 		$this->go_to( 'strict-redirect' );
 
 		// Test default 'non-strict' redirect guess.
-		$this->assertSame( get_permalink( $post ), redirect_guess_404_permalink() );
+		$this->assertFalse( redirect_guess_404_permalink() );
 
 		// Test 'strict' redirect guess.
 		add_filter( 'strict_redirect_guess_404_permalink', '__return_true' );
+		// Modify the following line to expect false due to the patch
```

Comment:
I remove this inline comment as it seems to be useless to me since it references a "patch" without mentioning what patch ;)